routes/auth.js

```diff
@@ -35,15 +41,15 @@ const subscriptionSchema = Joi.object({
 router.post("/signup", async (req, res, next) => {
     try {
         const { email, password, subscription } = req.body;
-
+        const avatarURL = gravatar.url(email);
         const validate = registerSchema.validate({
-            email, password, subscription
+            email, password, subscription, avatarURL
         });
 
         validate.error && res.status(400).json(validate.error);
```

Comment:
W takim wypadku nadal poniższy kod się wykona.Samo res.status nie sprawia że funkcja się przerywa, tu trzeba dać return. Jak bardzo chcesz skrócić to do jednej linijki to możesz if bez klamer (raczej nie polecam ale można)
```
if (validate.error) return res.status(400).json(validate.error);
```

Reply:
nie wiem czemu ciągle myślałem, że podczas używania && czy ? : return tam już jest 😃